express-get-json/index.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', (req, res) => {
+  for (const property in grades) {
+    res.json([grades]);
+  }
+});
```

Comment:
This sends a response with the wrong format. It sends an `Array` containing an `Object` containing 3 `Object`s, not an `Array` containing 3 `Object`s:

---

![image](https://user-images.githubusercontent.com/16143771/136621161-60e1c238-79c0-42ad-932b-36b67948ce9c.png)

---

![image](https://user-images.githubusercontent.com/16143771/136621049-8f7695e3-6422-4809-8531-d9c13cddc5e7.png)